jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarProcessors.java

```diff
@@ -116,6 +118,11 @@ static Integer determineJavaMajorVersion(Path jarPath) throws IOException {
             int majorVersion = classFile.readUnsignedShort();
             int javaVersion = (majorVersion - 45) + 1;
             return javaVersion;
+          } catch (EOFException ex) {
+            throw new IllegalArgumentException(
```

Comment:
Testing with a jar file containing a fake empty .class file  resulted in an EOFException with no error message. This change is to provide more clarity on why this exception is being thrown.

Reply:
Awesome. It's more likely that there's an empty `.class` file than a corrupted file.